src/webgpu/util/texture/texture_ok.ts

```diff
@@ -22,6 +22,13 @@ export type TexelCompareOptions = {
   maxDiffULPsForFloatFormat?: number;
 };
 
+export type PixelExpectation = PerTexelComponent<number> | Uint8Array;
+
+export type PerPixelComparison<E extends PixelExpectation> = {
+  location: GPUOrigin3D;
```

Comment:
not sure, but maybe `coord` instead of `location` ? seem to use "coord" in other parts of this PR

Reply:
Done